blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/credential/CredentialContainer.java

```diff
@@ -30,7 +32,9 @@
     private final Link self;
 
     public CredentialContainer() {
-        BlueOrganization organization= OrganizationFactory.getInstance().getContainingOrg(Jenkins.getInstance());
+        // seems like this is now getting common and should have a helper function
```

Comment:
@michaelneale there was No JIRA for this change but the tests where just broken and this fixes it.
I am not completely happy with this - but it is suitable for the 1.1 branch IMO - you may want to improve things when picked to the master branch.